backend/Utils/JWTUtils.ts

```diff
@@ -1,10 +1,24 @@
 import jwt from "jsonwebtoken";
+import { JWTModel } from "../DB/models/JWT";
 
 const { sign, verify } = jwt;
 
 export function makeJWT(payload: object): string {
   const secret = process.env.JWT_PASSWORD_RESET_SECRET || "secret";
 
   const token = sign(payload, secret, { expiresIn: "1h" });
+
+  const userId = payload["userId"];
+
+  saveJWT(token, userId);
+
   return token;
 }
+
+async function saveJWT(token: string, userId: number) {
+  const newJWT = await JWTModel.create({
+    token: token,
+    userId: userId,
+    jwtType: "passwordReset",
```

Comment:
The JWTUtils.ts file should not be specific to the reset password feature, since JWTs will be used in a variety of features. I think it would be best just to call JWTModel.create() inside of ResetPassword.ts rather than having a `saveJWT` function inside of JWTUtils - I don't think we need a wrapper function around JWTModel.create

Reply:
Yes!  Definitely want to make this reusable.  Thank you!